Add example systemd service and timer

mmilata opened the pull request:
Re #10, would systemd unit files help? They are based on files generated by NixOS where the paths are different, not sure where the binary/configuration would be on normal linux distro.

accumulator commented:
I've moved the files into contrib/ and added Type=oneshot.

The only thing I'm a bit worried about is that the paths currently used in the service file are non-typical (i.e. the installation instructions will install charge-lnd to /usr/local/bin or $HOME/.local/bin)

Could you add a # comment in the service file explaining to users to set the paths correctly for their situation?

mmilata commented:
Good point, changed the default paths and added some comments in dd32ead. Also removed `Description` since it doesn't do what it seems, according to the manual page.

accumulator commented:
Merged, thanks!
